Issue #103: fix codacy markdown findings in RELEASING.md

famod opened the pull request.

vackosar commented on RELEASING.md:
```diff
@@ -7,8 +7,9 @@ Note: Releases and SNAPSHOTs are deployed via [`nexus-staging-maven-plugin`](htt
+
 - **GPG**
-  - https://www.gnupg.org/download/
+  - [GnuPG download](https://www.gnupg.org/download/])
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/gitflow-incremental-builder/gitflow-incremental-builder/pullRequest?prid=4750698)

vackosar commented on RELEASING.md:
```diff
@@ -22,9 +23,9 @@ Note: Releases and SNAPSHOTs are deployed via [`nexus-staging-maven-plugin`](htt
+
 - **OSSRH** access
-  - https://central.sonatype.org/pages/ossrh-guide.html
+  - [OSSRH guide](https://central.sonatype.org/pages/ossrh-guide.html])
```
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/manual/gitflow-incremental-builder/gitflow-incremental-builder/pullRequest?prid=4750698)

famod commented:
Cleaned up, squashed, cherry-picked and directly pushed to `master`: https://github.com/vackosar/gitflow-incremental-builder/commit/8e554a6857bead80945ef69d4a7dc8f459d84ede
